<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>900</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>900</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>950</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_7">
   <item row="1" column="0">
    <widget class="QGroupBox" name="mcl_stage_groupBox">
     <property name="title">
      <string>MCL Piezo Stage</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="3" column="1">
       <widget class="QDoubleSpinBox" name="mcl_z_pos_doubleSpinBox"/>
      </item>
      <item row="3" column="2">
       <widget class="QDoubleSpinBox" name="mcl_z_target_doubleSpinBox"/>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="mcl_y_pos_doubleSpinBox"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>Z</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>X</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QDoubleSpinBox" name="mcl_x_target_doubleSpinBox"/>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Current</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Target</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="mcl_x_pos_doubleSpinBox"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>Y</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QDoubleSpinBox" name="mcl_y_target_doubleSpinBox"/>
      </item>
      <item row="3" column="3">
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Go To Center</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="mcl_connect_checkBox">
        <property name="text">
         <string>Connect</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3" rowspan="3">
       <widget class="PlotWidget" name="graphicsView">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="backgroundBrush">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_scanarea">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>150</height>
      </size>
     </property>
     <property name="title">
      <string>2D Scan Area</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="3" column="0">
       <widget class="QLabel" name="v_label">
        <property name="text">
         <string>y:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QDoubleSpinBox" name="v1_doubleSpinBox">
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="value">
         <double>45.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="h0_doubleSpinBox">
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="value">
         <double>25.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Dim. (H x V)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="h_label">
        <property name="text">
         <string>x:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>min</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QDoubleSpinBox" name="h1_doubleSpinBox">
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="value">
         <double>45.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QComboBox" name="scan_step_comboBox"/>
      </item>
      <item row="3" column="1">
       <widget class="QDoubleSpinBox" name="v0_doubleSpinBox">
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="value">
         <double>25.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QPushButton" name="center_pushButton">
        <property name="text">
         <string>Center on Pos</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Step Size</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QDoubleSpinBox" name="hspan_doubleSpinBox"/>
      </item>
      <item row="4" column="2">
       <widget class="QDoubleSpinBox" name="vspan_doubleSpinBox"/>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>max</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Power Meter</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_9">
      <item row="1" column="0">
       <widget class="QLabel" name="label_26">
        <property name="text">
         <string>Wavelength</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="3">
       <widget class="QLabel" name="power_meter_power_label">
        <property name="font">
         <font>
          <pointsize>24</pointsize>
         </font>
        </property>
        <property name="text">
         <string>0.000 W</string>
        </property>
        <property name="textFormat">
         <enum>Qt::PlainText</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QCheckBox" name="power_meter_acquire_cont_checkBox">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Acquire
Continuously</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="power_meter_wl_doubleSpinBox">
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="prefix">
         <string/>
        </property>
        <property name="suffix">
         <string> nm</string>
        </property>
        <property name="decimals">
         <number>0</number>
        </property>
        <property name="minimum">
         <double>100.000000000000000</double>
        </property>
        <property name="maximum">
         <double>3000.000000000000000</double>
        </property>
        <property name="value">
         <double>500.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QGroupBox" name="groupBox_scancontrol">
     <property name="title">
      <string>Spectrometer</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="3" column="0">
       <widget class="QLabel" name="label_23">
        <property name="text">
         <string>Center WL (nm)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="andor_ccd_int_time_doubleSpinBox">
        <property name="keyboardTracking">
         <bool>false</bool>
        </property>
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="minimum">
         <double>0.001000000000000</double>
        </property>
        <property name="maximum">
         <double>999.990000000000009</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QDoubleSpinBox" name="acton_spec_center_wl_doubleSpinBox">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="readOnly">
         <bool>false</bool>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="keyboardTracking">
         <bool>false</bool>
        </property>
        <property name="suffix">
         <string> nm</string>
        </property>
        <property name="minimum">
         <double>-100.000000000000000</double>
        </property>
        <property name="maximum">
         <double>9999.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_24">
        <property name="text">
         <string>Grating</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Exposure Time (s)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="andor_ccd_status_label">
        <property name="text">
         <string>STATUS</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="andor_ccd_temp_doubleSpinBox">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="suffix">
         <string> C</string>
        </property>
        <property name="decimals">
         <number>0</number>
        </property>
        <property name="minimum">
         <double>-300.000000000000000</double>
        </property>
        <property name="maximum">
         <double>999.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_22">
        <property name="text">
         <string>Temperature</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1" colspan="2">
       <widget class="QLineEdit" name="acton_spec_grating_lineEdit">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QComboBox" name="acton_spec_exitmirror_comboBox">
        <item>
         <property name="text">
          <string>EXIT MIRROR</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBox_stageposition">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>200</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>200</height>
      </size>
     </property>
     <property name="title">
      <string>ASI Stage Position</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0" rowspan="2" colspan="2">
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>xy</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="1" column="6">
          <widget class="QComboBox" name="xy_step_comboBox"/>
         </item>
         <item row="1" column="4">
          <widget class="QDoubleSpinBox" name="y_pos_doubleSpinBox"/>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="y_down_pushButton">
           <property name="text">
            <string>v</string>
           </property>
          </widget>
         </item>
         <item row="0" column="6">
          <widget class="QDoubleSpinBox" name="xy_step_doubleSpinBox"/>
         </item>
         <item row="1" column="3">
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>Y:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" rowspan="2">
          <widget class="QPushButton" name="x_down_pushButton">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>&lt;</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="y_up_pushButton">
           <property name="text">
            <string>^</string>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QDoubleSpinBox" name="x_pos_doubleSpinBox"/>
         </item>
         <item row="0" column="2" rowspan="2">
          <widget class="QPushButton" name="x_up_pushButton">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>&gt;</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>X:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="5">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>xy step:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="6">
          <widget class="QPushButton" name="stop_stage_pushButton">
           <property name="text">
            <string>HALT</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QGroupBox" name="asi_xy_target_groupBox">
        <property name="title">
         <string>XY Target</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="3">
          <widget class="QDoubleSpinBox" name="asi_y_target_doubleSpinBox"/>
         </item>
         <item row="0" column="1">
          <widget class="QDoubleSpinBox" name="asi_x_target_doubleSpinBox"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>x:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>y:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QGroupBox" name="power_wheel_groupBox">
     <property name="title">
      <string>Power Wheel</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_16" columnstretch="0,0,0,0,0">
      <property name="sizeConstraint">
       <enum>QLayout::SetMinimumSize</enum>
      </property>
      <property name="horizontalSpacing">
       <number>0</number>
      </property>
      <item row="1" column="3">
       <widget class="QPushButton" name="powerwheel_move_fwd_pushButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>10</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>+</string>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QDoubleSpinBox" name="powerwheel_move_steps_doubleSpinBox"/>
      </item>
      <item row="1" column="4">
       <widget class="QPushButton" name="powerwheel_move_bkwd_pushButton">
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label_25">
        <property name="text">
         <string>Move Delta</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" colspan="3">
       <widget class="QDoubleSpinBox" name="power_wheel_encoder_pos_doubleSpinBox"/>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_29">
        <property name="text">
         <string>Encoder</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="6" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>h0_doubleSpinBox</tabstop>
  <tabstop>h1_doubleSpinBox</tabstop>
  <tabstop>v0_doubleSpinBox</tabstop>
  <tabstop>v1_doubleSpinBox</tabstop>
  <tabstop>hspan_doubleSpinBox</tabstop>
  <tabstop>vspan_doubleSpinBox</tabstop>
  <tabstop>scan_step_comboBox</tabstop>
  <tabstop>center_pushButton</tabstop>
  <tabstop>mcl_connect_checkBox</tabstop>
  <tabstop>mcl_x_pos_doubleSpinBox</tabstop>
  <tabstop>mcl_y_pos_doubleSpinBox</tabstop>
  <tabstop>mcl_z_pos_doubleSpinBox</tabstop>
  <tabstop>mcl_x_target_doubleSpinBox</tabstop>
  <tabstop>mcl_y_target_doubleSpinBox</tabstop>
  <tabstop>mcl_z_target_doubleSpinBox</tabstop>
  <tabstop>graphicsView</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>x_down_pushButton</tabstop>
  <tabstop>y_up_pushButton</tabstop>
  <tabstop>y_down_pushButton</tabstop>
  <tabstop>x_up_pushButton</tabstop>
  <tabstop>x_pos_doubleSpinBox</tabstop>
  <tabstop>y_pos_doubleSpinBox</tabstop>
  <tabstop>xy_step_doubleSpinBox</tabstop>
  <tabstop>xy_step_comboBox</tabstop>
  <tabstop>stop_stage_pushButton</tabstop>
  <tabstop>asi_xy_target_groupBox</tabstop>
  <tabstop>asi_x_target_doubleSpinBox</tabstop>
  <tabstop>asi_y_target_doubleSpinBox</tabstop>
  <tabstop>power_wheel_encoder_pos_doubleSpinBox</tabstop>
  <tabstop>powerwheel_move_steps_doubleSpinBox</tabstop>
  <tabstop>powerwheel_move_fwd_pushButton</tabstop>
  <tabstop>powerwheel_move_bkwd_pushButton</tabstop>
  <tabstop>power_meter_wl_doubleSpinBox</tabstop>
  <tabstop>power_meter_acquire_cont_checkBox</tabstop>
  <tabstop>andor_ccd_int_time_doubleSpinBox</tabstop>
  <tabstop>acton_spec_center_wl_doubleSpinBox</tabstop>
  <tabstop>acton_spec_exitmirror_comboBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
